packages/xchain-ethereum/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v.0.22.5 (2021-09-09)
```

Comment:
any reason we jumped  from 0.22.3 -> 0.22.5? not a huge deal anyway

Reply:
nevrmind i saw that the version was 0.22.4 in package.json

Reply:
Good point ^ ! Is there a way to find out changes of `0.22.4` to add these to CHANGELOG?